src/connectObs.js

```diff
@@ -40,8 +40,9 @@ const checkObservable = (observable, name) => {
 
 const aggregateProps = values => values.reduce((acc, value) => ({...acc, ...value}));
 
-const connectObs = mapObservables => withObs((observables) => {
-  const obsMap = mapObservables(observables);
+const connectObs = obsMapper => withPropsStream((props$, obs) => {
+  // TODO: change the interface of obsMapper. Should be `obsMapper(obs, props$)`.
```

Comment:
If we can agree that the right interface for `mapObs`'s mapper is `(obs, $props) => obs` then I think it makes sense for `connectObs`'s mapper too.